### PR TITLE
Perform some docs smithing

### PR DESCRIPTION
**Pull request by mavam**
In this round of docs smithing, we're improving some setup and user guides.

### CODE REVIEW

**Comment by mavam** [comment]
@dominiklohmann I think it's still better to keep node sizing and deployment separate. Sizing is really a paper exercise and unrelated to the installation steps.

**Comment by dominiklohmann** [comment]
Fine by me, I don't have a too strong opinion on that.

File: web/docs/user-guides/manage-a-pipeline/app-controls.png
Comment (dominiklohmann) [inline web/docs/user-guides/manage-a-pipeline/app-controls.png:1]: Please run this through TinyPNG or similar, replace this image and then force-push. Uploading this to TinyPNG I got a 69.5% reduction in file size with a negligible loss in quality.

File: web/docs/user-guides/run-pipelines/README.md [hunk 1]
```diff
@@ -68,3 +68,26 @@ tenzir -f pipeline.tql
...
+
+```yaml {0} title="<prefix>/etc/tenzir/tenzir.yaml"
+pipelines:
+  suricata-over-tcp:
+    name: Import Suricata from TCP
+    definition: |
+      from tcp://0.0.0.0:34343 read suricata
+      | import
+    start:
+      failed: true  # always restart on failure
```
Comment (dominiklohmann) [inline web/docs/user-guides/run-pipelines/README.md:92]: Should we list all possible options here?

**Review by dominiklohmann — APPROVED** [review]
This reads nicely, just one request to keep the PNG sizes down and one small review comment. Approving already.

File: web/docs/user-guides/run-pipelines/README.md (hunk 1 above)
Comment (mavam) [reply web/docs/user-guides/run-pipelines/README.md:92]: Yes, what are they exactly?

Comment (mavam) [reply web/docs/user-guides/run-pipelines/README.md]: @dominiklohmann please take another look and resolve if you're fine with the current state.

Comment (mavam) [reply web/docs/user-guides/run-pipelines/README.md]: @tobim you may want to peek at the docs and sanity check them with your implementation.

Comment (dominiklohmann) [reply web/docs/user-guides/run-pipelines/README.md]: I think that's all, yes.

**Pull request merged**